### PR TITLE
A simple Components page

### PR DESCRIPTION
**Pull request by CamLamb**
dwds-components

### CODE REVIEW

File: src/dw_design_system/templates/dw_design_system/components.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+{% endblock page_title %}
+
+{% block primary_content %}
+    {% for component in components %}
+        {% include "dw_design_system/partials/component.html" %}
+    {% endfor %}
+{% endblock primary_content %}
+
+{% block scripts %}
+    <style type="text/css">
```
Comment (marcelkornblum) [inline src/dw_design_system/templates/dw_design_system/components.html:19]: ought this to be a css file?

File: src/dw_design_system/utils.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+from wagtail.images.models import Image
+
+
+DATETIME_STR = "datetime"
+IMAGE_STR = "Image"
+
+
+def get_components():
+    thumbnail_file = Image.objects.first()
+    return [
```
Comment (marcelkornblum) [inline src/dw_design_system/utils.py:14]: not a biggie but for me this being a data file would be preferable; easier to maybe run a scan and check it has the right number of elements in it or whatever

**Review by marcelkornblum — APPROVED** [review]
this is very smart, i like it :)

File: src/dw_design_system/templates/dw_design_system/components.html (hunk 1 above)
Comment (CamLamb) [reply src/dw_design_system/templates/dw_design_system/components.html:19]: Yeah, we can move it to a separate CSS file in a separate PR. I'm going to merge this for now so I can unblock people 😄

File: src/dw_design_system/utils.py (hunk 2 above)
Comment (CamLamb) [reply src/dw_design_system/utils.py:14]: Added a todo :)

**Pull request merged**